compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -5690,7 +5749,7 @@ private BType getErrorType(BType collectionType, BLangQueryExpr queryExpr, Analy
         return errorType;
     }
 
-    private BType getNonContextualQueryType(BType staticType, BType basicType) {
+    private BType getNonContextualQueryType(BLangQueryExpr queryExpr, BType staticType, BType basicType) {
```

Comment:
It is not needed to pass `queryExpr` now.

Reply:
Will update it. Thanks.